### PR TITLE
Updates for the sorting page

### PR DESCRIPTION
**Pull request by rsill-neo4j**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-173.surge.sh

Updated pages:
- [Sorting](https://neo4j-docs-graphql-173.surge.sh/graphql/5/queries-aggregations/sorting)

**Review by lidiazuin — APPROVED** [review]
LGTM!

**Review by mjfwebb — APPROVED** [review]

File: modules/ROOT/pages/queries-aggregations/sorting.adoc [hunk 1]
```diff
@@ -16,7 +16,7 @@ type Movie {
 }
 ----
 
-The following sorting input type and query should be generated:
+The following sorting input type and query is generated:
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/queries-aggregations/sorting.adoc]:
```suggestion
The following sorting input type and query are generated:
```

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**